Source/WebCore/platform/mock/MockMediaDevice.h

```diff
@@ -247,9 +252,9 @@ struct MockMediaDevice {
         if (!label)
             return std::nullopt;
 
-        std::optional<bool> isEphemeral;
-        decoder >> isEphemeral;
-        if (!isEphemeral)
+        std::optional<Flags> flags;
```

Comment:
```suggestion
        std::optional<OptionSet<Flags>> flags;
```
But probably Flags should be defined as:
`using Flags = OptionSet<Flag>`

Reply:
Done